CRAN_questions.md

```diff
@@ -0,0 +1,19 @@
+This document briefly presents some questions this working group has discussed about CRAN to help it being successful.
+
+ * CRAN does an amazing amount of work. What are the most critical issues that CRAN is facing?
+ * CRAN has worked relatively similar for the first 20 years. What is the future of CRAN for the next 20 years?
```

Comment:
I would avoid making any judgements about how CRAN has worked. The statement of it working similarly implies that they may be stuck in a rut. I would instead phrase this as "CRAN has been extraordinarily successful over its first 20 years; how can we ensure that its next 20 years are just as successful?".